File: calculator.kt
fun main() {
    val input = readLine()!!.split(" ")
    val num1 = input[0].toDouble()
    val operator = input[1]
    val num2 = input[2].toDouble()

    val result = when (operator) {
        "+" -> num1 + num2
        "-" -> num1 - num2
        "*" -> num1 * num2
        "/" -> num1 / num2
        else -> throw IllegalArgumentException("Invalid operator")
    }

    println(result)
}

File: calculator.cs
using System;

class Program {
    static void Main() {
        string[] input = Console.ReadLine().Split(' ');
        double num1 = Convert.ToDouble(input[0]);
        string operatorSymbol = input[1];
        double num2 = Convert.ToDouble(input[2]);

        double result = operatorSymbol switch {
            "+" => num1 + num2,
            "-" => num1 - num2,
            "*" => num1 * num2,
            "/" => num1 / num2,
            _ => throw new InvalidOperationException("Invalid operator")
        };

        Console.WriteLine(result);
    }
}
